matplotlibcpp.h

```diff
@@ -68,6 +68,14 @@ namespace matplotlibcpp {
 			}
 
 			private:
+#if PY_MAJOR_VERSION >= 3
+				int init_numpy(){
+#else
+				void init_numpy(){
+#endif
+					import_array()
```

Comment:
Hi, sorry for the late review! The rest of the PR looks great, but I think this is missing a `return 0;` in the Python 3 case. It would be great if you could amend and force push this.